subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/runtime/Runtime.kt

```diff
@@ -34,17 +33,21 @@ fun extensionOf(target: Any, extensionName: String): Any =
     (target as ExtensionAware).extensions.getByName(extensionName)
 
 
+@Deprecated("The concept of conventions is deprecated. Use extensions if possible.")
```

Comment:
❓ this type is internal and its members are invoked by generated bytecode only, I would not add `@Deprecated` to those three functions

Reply:
Makes sense, thanks for the insight.